installer/roles/kubernetes/templates/deployment.yml.j2

```diff
@@ -487,9 +487,13 @@ metadata:
   labels:
     name: {{ kubernetes_deployment_name }}-web-svc
 spec:
+  type: {{ kubernetes_web_svc_type }}
   ports:
     - name: http
       port: 80
+{% if kubernetes_web_svc_type == "ClusterIP" %}
```

Comment:
necessary in order to go from type NodePort to ClusterIP

see https://github.com/kubernetes/kubectl/issues/221

and in particular this comment

https://github.com/kubernetes/kubectl/issues/221#issuecomment-514173913